lib/active_merchant/billing/credit_card.rb

```diff
@@ -158,6 +158,21 @@ def brand=(value)
       # @return [String]
       attr_accessor :fallback_reason
 
+      # Returns or sets whether card-present card data has been read contactlessly.
+      #
+      # @return [true, false]
+      attr_accessor :contactless
+
+      # Returns the ciphertext of the card's encrypted PIN.
+      #
+      # @return [String]
+      attr_accessor :encrypted_pin
+
+      # Returns the Key Serial Number (KSN) of the card's encrypted PIN.
+      #
+      # @return [String]
+      attr_accessor :encrypted_pin_key_id
```

Comment:
So a credit card carries on it an ID rather than a KSN? The acquiring bank knows what to do with it?